src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -196,18 +197,19 @@ export const ContentMessageComponent: React.FC<ContentMessageProps> = ({
             message={message}
             is1to1Conversation={conversation.is1to1()}
             isLastDeliveredMessage={isLastDeliveredMessage}
-            onClick={onClickDetails}
```

Comment:
We shouldn't be able to click the read icon next to the username anymore?
In that case `cursor: pointer` is wrongly assigned to that element